test/bpk-token-fail.jsx

```diff
@@ -1,4 +1,3 @@
-import React from 'react';
```

Comment:
We want this file to fail, but not for this reason, so removing this to ensure that this is _only_ failing for the reason we expect

Reply:
There is no usage in this file, so it just isn't need (unless I've misunderstood your request)?